tests/acceptance/characters/test_ursula_web_status.py

```diff
@@ -41,9 +41,11 @@ def test_ursula_html_renders(ursula, client):
     assert response.status_code == 404
     response = client.get('/status/')
     assert response.status_code == 200
+    with open("status.html", "wb") as f:
+        f.write(response.data)
```

Comment:
My bad, testing stuff. Will remove that.

Reply:
BTW, I checked that using the `federated_ursulas` fixture instead of `blockchain_ursulas` works equally well for this test, but it's waaaay faster.

Reply:
Good idea! Still requires an explicit call to `learn_from_teacher_node()` to propagate fleet states, but yeah, works much faster. I don't want to over-complicate this PR, so I'll put this change into the one I'm building on top of it.

Reply:
Fixed.